src/Hosting/test/FunctionalTests/ShutdownTests.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Hosting.IntegrationTesting;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Hosting.FunctionalTests
+{
+    public class ShutdownTests : LoggedTest
+    {
+        private static readonly string StartedMessage = "Started";
+        private static readonly string CompletionMessage = "Stopping firing\n" +
+                                                            "Stopping end\n" +
+                                                            "Stopped firing\n" +
+                                                            "Stopped end";
+
+        public ShutdownTests(ITestOutputHelper output) : base(output) { }
```

Comment:
I don't think this is necessary, try deleting this and see if the logs are still being written.